### PR TITLE
Ziti webhook for dark Jenkins

### PR DESCRIPTION
**Pull request by qrkourier**
I'll test the end-to-end with this branch then mark the PR ready for review.

### CODE REVIEW

**Review by qrkourier — COMMENTED** [review]
waiting for release tag `v1` to become available on webhook action [after PR merges](https://github.com/openziti/ziti-webhook-action/pull/10).

**Review by dovholuknf — APPROVED** [review]

**Pull request merged**
